packages/react-components/react-textarea/stories/Textarea/index.stories.tsx

```diff
@@ -11,7 +11,7 @@ export { Resize } from './TextareaResize.stories';
 export { Size } from './TextareaSize.stories';
 export { Uncontrolled } from './TextareaUncontrolled.stories';
 export { Controlled } from './TextareaControlled.stories';
-export { Height } from './TextareaHeight.stories';
```

Comment:
Was this a mistake to delete the `Height` story? If you meant to delete it, then also delete TextareaHeight.stories.tsx